demo/ContainerViewController.swift

```diff
@@ -61,6 +62,20 @@ final class ContainerViewController: UIViewController {
             distanceLabel.heightAnchor.constraint(equalToConstant: buttonHeight),
         ])
         
+        view.addSubview(collisionAlertView)
+        NSLayoutConstraint.activate([
+            collisionAlertView.topAnchor.constraint(equalTo: view.topAnchor, constant: bannerInset),
+            collisionAlertView.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+        ])
+        
+        view.addSubview(collisionBanerView)
+        NSLayoutConstraint.activate([
+            collisionBanerView.topAnchor.constraint(equalTo: view.topAnchor),
```

Comment:
Is it needed to resize the banner to full screen? Maybe centering the image will do?

Reply:
This is because of dark background. Got it.